server/RequestHandler.ts

```diff
@@ -1,19 +1,16 @@
-import * as express from "express";
 import * as fs from "fs-extra";
 import * as eta from "../eta";
 import Application from "./Application";
 
 /**
  * Logic for processing HTTP requests. Instantiated per request.
  */
-export default class RequestHandler extends eta.IRequestHandler {
-    public route: string;
-    public action: string;
-    public routeParams: {[key: string]: string};
-    public controller: eta.IHttpController;
-    public controllerPrototype: eta.IHttpController;
+export default class RequestHandler extends eta.RequestHandler {
```

Comment:
Will there be any potential confusion with having the class and interface named the same? Also, if the interface below is still named IRequestHandler idk if `eta generate` just exports it without the `I` but you might need to be sure